pages.pt_BR/common/cmatrix.md

```diff
@@ -0,0 +1,20 @@
+# cmatrix
+
+> Exibe um padrão semelhante à Matrix rolando na tela do terminal.
+> Mais informações: <https://github.com/abishekvashok/cmatrix>.
+
+- Habilitar rolagem assincrona:
+
+`cmatrix -a`
+
+- Exibir texto em vermelho:
+
+`cmatrix -C {{red}}`
+
+- Habilitar modo arco-iris:
+
+`cmatrix -r`
+
+- Determina um atraso de 2 centsegundos (20 milisegundos) na atualização da tela:
```

Comment:
`determina` fits better than `define` or `estabelece`?
```suggestion
- Determina um atraso de 2 centisegundos (20 milisegundos) na atualização da tela:
```

Reply:
I believe that they all fit in this situation. So, it's a matter of preference.

Reply:
Yes, there was a typo. It's already fixed.